BandalArt/Projects/Core/Network/Sources/Infra/BandalArtAPIConstant.swift

```diff
@@ -0,0 +1,13 @@
+//
+//  BandalArtAPIConstant.swift
+//  Network
+//
+//  Created by Sang hun Lee on 2023/07/27.
+//  Copyright © 2023 Otani. All rights reserved.
+//
+
+import Foundation
+
+struct BandalArtAPIConstant {
```

Comment:
별거 아닌데  
```suggestion
struct APIConstant {
```
으로 가눈거 오땨?

Reply:
좋아유 ㅋㅋㅋ 혹시나 다른 타겟으로 API 쓸 일은 없겠지..?